include/mamba/api/configuration.hpp

```diff
@@ -18,6 +18,33 @@
 #include <functional>
 
 
+#ifdef ENABLE_CONTEXT_DEBUG_PRINT
+#define CONTEXT_DEBUGGING                                                                          \
+    if (Configuration::instance().at("print_context_only").value<bool>())                          \
+    {                                                                                              \
+        Context::instance().debug_print();                                                         \
+        exit(0);                                                                                   \
+    }
+#define CONFIG_DEBUGGING                                                                           \
```

Comment:
this is defined also a couple of lines below

Reply:
sorry. misread!

Reply:
:)

Reply:
:)